cassandra-atomicapp/Nulecule

```diff
@@ -0,0 +1,53 @@
+---
+specversion: 0.0.2
+id: cassandra-atomicapp
+
+metadata:
+  name: Cassandra App
+  appversion: 0.0.1
+  description: Cassandra Atomic App
+
+graph:
+  - name: cassandra-app
+    params:
+      - name: image
+        description: The cassandra image
+      - name: hostport
+        description: The port where cassandra runs
+        default: 7000
+      - name: listen_address
+        description: This variable is for controlling which IP address to listen for incoming connections on.
+        default: auto
+      - name: broadcast_address
+        description: This variable is for controlling which IP address to advertise to other nodes.
+        default: auto
+      - name: rpc_address
+        description: This variable is for controlling which address to bind the thrift rpc server to.
+        default: 0.0.0.0
+      - name: start_rpc
+        description: This variable is for controlling if the thrift rpc server is started.
```

Comment:
Don't need `This variable is for controlling` in each description. It'll be cleaner to remove it and just say "If the thrift rpc server is started"

Same goes with "Which IP address to advertise to other nodes"